Dockerfile

```diff
@@ -26,9 +30,17 @@ RUN apt-get update && apt-get install -y git \
 
 RUN useradd --create-home node
 
-ENTRYPOINT ["java", "-jar", "/git-bridge.jar"]
-CMD ["/conf/runtime.json"]
-
 COPY --from=builder /git-bridge.jar /
 
+COPY vendor/envsubst /opt/envsubst
+RUN chmod +x /opt/envsubst
+
+COPY conf/envsubst_template.json envsubst_template.json
+COPY start.sh start.sh
+
+RUN mkdir conf
+RUN chown node:node conf
+
 USER node
+
+ENTRYPOINT ["/start.sh"]
```

Comment:
One comment on this: We could leave `ENTRYPOINT` as-is and swap this out for `CMD`. I think `ENTRYPOINT` might lean towards the preferred way of doing things, but it's less consistent with how we run other apps in production.

In particular, we tend to run `kubectl exec <podname> bash` to get into the container and debug what's going on - which IIRC passes the command to the entrypoint.

Reply:
Done.